server/base/src/test/java/org/apache/accumulo/server/manager/state/RootTabletStateStoreTest.java

```diff
@@ -92,6 +93,10 @@ public void mutate() {
   public void testRootTabletStateStore() throws DistributedStoreException {
     ServerContext context = MockServerContext.get();
     expect(context.getAmple()).andReturn(new TestAmple()).anyTimes();
+    expect(context.getConfiguration().isPropertySet(Property.TSERV_LAST_LOCATION_MODE))
+        .andReturn(true).once();
+    expect(context.getConfiguration().get((Property.TSERV_LAST_LOCATION_MODE)))
+        .andReturn(Property.TSERV_LAST_LOCATION_MODE.getDefaultValue()).once();
```

Comment:
I looked at a little bit and I think the following changes could improve the situation, but not sure if its workable.

 * Add a metthod called `MockServerContext.create(Map<String,String> props)` that creates a server context with the requested system props.
 * Add a TestAmple constructor that takes server context. 

Then may be able to do the following.

```suggestion
  public void testRootTabletStateStore() throws DistributedStoreException {
    ServerContext context = MockServerContext.create(Map.of(Property.TSERV_LAST_LOCATION_MODE, Property.TSERV_LAST_LOCATION_MODE.getDefaultValue()));
    expect(context.getAmple()).andReturn(new TestAmple(context)).anyTimes();
```